docs/help/support-guidelines.md

```diff
@@ -1,5 +1,86 @@
 # Support guidelines
 
+This page explains the support mechanisms we provide.
+
 !!! warning
 
-    Under Construction
+    Before asking any help, search and read this documentation site carefully.
+    Also, understand and follow our [discussion guidelines](../contributing/discussion-guidelines/index.md) during your discussions.
+
+Choose appropriate resources depending on what kind of help you need:
+
+- Documentation sites
+  - Various information
+- GitHub Discussions
+  - Questions
+  - Unconfirmed bugs
+  - Feature requests
+  - Design discussions
+- GitHub Issues
+  - Confirmed bugs
+- Slack
+  - Instant messaging between contributors
+- ROS Discourse
+  - General topics that should be widely announced
+
+The following sections are detailed descriptions of each resource.
+
+## Documentation sites
+
+[Docs guide](docs-guide.md) shows the list of useful documentation sites.
+Visit them and see if there is any information related to your problem.
+
+Note that the documentation sites aren't always latest and perfect.
+If you find some information is wrong, unclear, or missing in Autoware docs, feel free to submit a pull request following our [contribution guidelines](../contributing/index.md).
+
+!!!warning
+
+    Since this documentation site is still under construction, there are some empty pages.
+
+## GitHub Discussions
+
+If you have a problem with Autoware, check existing issues and questions and search similar issues first.
+
+- [Issues](https://github.com/autowarefoundation/autoware/issues)
+
+  Note that Autoware has multiple repositories listed in [autoware.repos](https://github.com/autowarefoundation/autoware/blob/main/autoware.repos).  
+  It is recommended to search across the repositories.
+
+- [Questions](https://github.com/autowarefoundation/autoware/discussions/categories/q-a)
+
+If no answer was found, create a new question thread [here](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
+If your question is not answered in a timely manner (about a week), mention maintainers and remind them.
+
+Also, there are other discussion types such as [feature requests](https://github.com/autowarefoundation/autoware/discussions/categories/feature-requests) or [design discussions](https://github.com/autowarefoundation/autoware/discussions/categories/design).
+Feel free to open or join such discussions.
+
+## GitHub Issues
+
+If you have a problem and you have confirmed it is a bug, find the appropriate repository and create a new issue there.
+If you can't determine the appropriate repository, you should not create an issue yet.
```

Comment:
```suggestion
If you can't determine the appropriate repository, please ask about where to create it in the [Q&A section](https://github.com/autowarefoundation/autoware/discussions/categories/q-a). And the maintainers will help you decide.
```

Reply:
- Please remove `please`.
- `Q&A section` -> `Q&A category` is better?
- I'm not sure whether I should add `the` for `maintainers`.
    - Will follow the review result of https://github.com/autowarefoundation/autoware-documentation/pull/40/files/ae7fe87953f66dff58bde36fdb1568d82972ef43#r822744588.

Reply:
`category` is better.

removed `please`.

Reply:
Also, I wonder if I can use `can't` or `don't` instead of `cannot` or `do not` for technical documents.
Seeing https://developers.google.com/style?hl=en, it's allowed?

But I'd like to hear @LalithVipulananthan's comment to clarify the rule.

Reply:
You can of course use, no ome can argue against that.

Reply:
I have to admit that I am not sure about what is acceptable for a modern technical guide with respect to `can't` or `don't` instead of `cannot` or `do not`, so would defer to the Google developer documentation style guide in this case.

My alternative suggestion for the line above:

```suggestion
If you can't determine the appropriate repository, ask the maintainers for help by creating a new Discussion in the [Q&A category](https://github.com/autowarefoundation/autoware/discussions/categories/q-a).
```

Reply:
Thank you!
I don't stick to small differences, but could I ask some questions so that I can properly review other developers in the future?

- Should I add `the` for `maintainers` here?
- Why did you capitalize `Discussion`?

Reply:
- In the sentence above, it should be `ask the maintainers` rather than `ask maintainers`. Does that answer your question, or was there a different example that you wanted me to clarify?
- For `Discussion`, I capitalized it because the feature is referred to as [GitHub Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). Then I read that documentation and noticed that they don't capitalize the word when referring to individual discussions so it should be `discussion` instead.